src/api/roles/client/telemetry.py

```
@@ -140,17 +145,348 @@ def query_workouts(
+
+    # ── Calories burned (workouts) ────────────────────────────────────
+    workout_rows = db.exec(
+        select(CompletedWorkout, ClientTelemetry, CompletedWorkoutActivity)
+        .join(ClientTelemetry, CompletedWorkout.client_telemetry_id == ClientTelemetry.id)
+        .join(CompletedWorkoutActivity, CompletedWorkoutActivity.id == CompletedWorkout.completed_workout_details_id, isouter=True)
+        .where(ClientTelemetry.client_id == client_id)
+    ).all()
+
+    burned = 0.0
+    workout_count = 0
+    for completed, telemetry, details in workout_rows:
```

Ruff (0.15.12)

[warning] Loop control variable `completed` not used within loop body

Rename unused `completed` to `_completed`

(B007)